SQL/New_patches/2018-09-07_EEG_tables.sql

```diff
@@ -0,0 +1,259 @@
+-- Create a physiological_modality table
+CREATE TABLE `physiological_modality` (
+  `PhysiologicalModalityID` TINYINT UNSIGNED NOT NULL AUTO_INCREMENT,
+  `PhysiologicalModality`   VARCHAR(50)      NOT NULL,
+  PRIMARY KEY (`PhysiologicalModalityID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+-- Create a physiological_output_type table
+CREATE TABLE `physiological_output_type` (
+  `PhysiologicalOutputTypeID` INT(5) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `OutputType`                VARCHAR(20)     NOT NULL,
+  `OutputTypeDescription`     VARCHAR(255),
+  PRIMARY KEY (`PhysiologicalOutputTypeID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+-- Create a physiological_file table
+CREATE TABLE `physiological_file` (
+  `PhysiologicalFileID`       INT(10)    UNSIGNED  NOT NULL      AUTO_INCREMENT,
+  `PhysiologicalModalityID`   TINYINT(3) UNSIGNED  DEFAULT NULL,
+  `Caveat`                    TINYINT(1) UNSIGNED  DEFAULT 0,
+  `PhysiologicalOutputTypeID` INT(5)     UNSIGNED  NOT NULL,
+  `SessionID`                 INT(10)    UNSIGNED  NOT NULL,
+  `InsertTime`                INT(10)    UNSIGNED  NOT NULL,
+  `FileType`                  VARCHAR(12)          DEFAULT NULL,
+  `AcquisitionTime`           VARCHAR(20)          DEFAULT NULL,
+  `InsertedByUser`            VARCHAR(50)          NOT NULL,
```

Comment:
Is this a Loris user? if yes, UserID would be better

Reply:
Can you add a comment to that effect in the SQL file? It's not ideal, but I don't know where else to put it and it's a little confusing.

Reply:
sure. Will do in the next commit!